addons/drawbuild/functions/fnc_drawBuildZeus.sqf

```diff
@@ -2,104 +2,92 @@
 
-			// close dialog
-			closeDialog 1;
-
-			// get options for simulation and damage 
-			private _ctrlSimCheckbox = _display displayCtrl IDC_CHECKBOX_SIMULATION;
-			private _ctrlDmgCheckbox = _display displayCtrl IDC_CHECKBOX_DAMAGE;
-
-			private _enableSim = cbChecked _ctrlSimCheckbox;
-			private _enableDmg = cbChecked _ctrlDmgCheckbox;
-
-			// start draw building
-			[_object, _enableSim, _enableDmg] call FUNC(drawBuildSelectPosition);
-			
-    }, [_mainItem, _display]] call CBA_fnc_addBISEventHandler;
-
-} forEach _arrOptions;
+private _objects = [];
+private _prettyNames = [];
+{
+    private _displayName = getText (configfile >> "CfgVehicles" >> _x >> "displayName");
+    private _picture = getText (configFile >> "CfgVehicles" >> _x >> "editorPreview");
+    if !(fileExists _picture) then { _picture = getText (configFile >> "CfgVehicles" >> _x >> "icon") };
+    if !(fileExists _picture) then { _picture = getText (configFile >> "CfgVehicles" >> _x >> "picture") };
+
+    _objects pushBack _x;
+    _prettyNames pushBack [_displayName, "", _picture];
+
+} forEach _drawPresets;
+
+
+// Get the offset of the first (alphabetical) object, to populate the default for the appropriate field
+private _firstObject = ([keys GVAR(drawBuildPresets), [], { getText(configfile >> "CfgVehicles" >> _x >> "displayName") }] call BIS_fnc_sortBy)#0;
+private _initialOffset = str((GVAR(drawBuildPresets) get _firstObject)#0);
+
+[
+	localize "STR_CROWSZA_Drawbuild_module_name", 
+	[
+		["EDIT",localize "STR_CROWSZA_Drawbuild_filter",["", {
+			private _filter = _this;
+			private _filteredObjects = (keys GVAR(drawBuildPresets) select { ([_filter, getText(configfile >> "CfgVehicles" >> _x >> "displayName")] call BIS_fnc_inString) });
+			_filteredObjects = [_filteredObjects, [], { getText(configfile >> "CfgVehicles" >> _x >> "displayName") }] call BIS_fnc_sortBy;
+
+			private _listbox = (allControls (uiNamespace getVariable "zen_common_display")) select { ctrlType _x == 5} select 0;
+			lbClear _listbox;
+			private _index = 0;
+			{
+				_listbox lbAdd getText(configfile >> "CfgVehicles" >> _x >> "displayName");
+                private _picture = getText (configFile >> "CfgVehicles" >> _x >> "editorPreview");
+                if !(fileExists _picture) then { _picture = getText (configFile >> "CfgVehicles" >> _x >> "icon") };
+                if !(fileExists _picture) then { _picture = getText (configFile >> "CfgVehicles" >> _x >> "picture") };
+			    _listbox lbSetPicture [_index, _picture];
+			    _listbox setVariable [str _index, _x];
+			    _index = _index+1;
+			} forEach _filteredObjects;
+
+			call FUNC(updateDrawBuildUI);
+
+			_this
+		}], true],
+		["LIST",localize "STR_CROWSZA_Drawbuild_objects_to_build",[_objects, _prettyNames, 0, 15]],
+		["EDIT",[localize "STR_CROWSZA_Drawbuild_custom_object",(localize "STR_CROWSZA_Drawbuild_custom_object_tooltip") regexReplace ["<br/>",endl]],["", {}]],
```

Comment:
Were you not able to do this instead of regex?
```sqf
["EDIT",[localize "STR_CROWSZA_Drawbuild_custom_object",parseText localize "STR_CROWSZA_Drawbuild_custom_object_tooltip"],["", {}]],
```

Like I have done with: 
```sqf
_controls pushBack ["SLIDER", [localize "STR_CROWSZA_Misc_strip_explosives_leave_untouched", parseText localize "STR_CROWSZA_Misc_strip_explosives_leave_untouched_tooltip"], [0, 10, _leave, 0], _default];
```
https://github.com/Crowdedlight/Crows-Zeus-Additions/blob/develop/addons/misc/functions/fnc_constructstripExplosivesDialog.sqf#L142